src/sounds.cpp

```diff
@@ -160,6 +161,34 @@ static std::vector<centroid> cluster_sounds( std::vector<std::pair<tripoint, int
     return sound_clusters;
 }
 
+int get_signal_for_hordes( const centroid &centr )
+{
+    //Volume in  tiles. Signal fo hordes in submaps
+    const int vol = centr.volume - weather_data( g->weather ).sound_attn; //modify vol using weather vol.Weather can reduce monster hearing
+    const int min_vol_cap = 60;//Hordes can't hear volume lower than this
+    const int undeground_div = 2;//Coeffficient for volume reduction undeground
+    const int hordes_sig_div =  SEEX;//Divider coefficent for hordes
+    const int min_sig_cap = 8; //Signal for hordes can't be lower that this if it pass min_vol_cap
+    const int max_sig_cap = 26;//Signal for hordes beyond this cap will be reduced heavily
+    //Balancing coefs:
+    const float sig_coef = 1;// To make player life easier - make it less than one. To make harder - more than  one
+    const float sig_modifier = 0;// To make player life easier - make it less than zero. To make harder - more than zero
+
+    //Lower the level- lower the sound
+    int vol_hordes = ( ( centr.z < 0 ) ? vol / ( undeground_div * std::abs( centr.z ) ) : vol );
+    if( vol_hordes > min_vol_cap ) {
+        //Calculating horde hearing signal
+        int sig_power = std::ceil( ( float ) vol_hordes / hordes_sig_div * sig_coef + sig_modifier );
+        //Capping minimum horde hearing signal
+        sig_power = ( ( sig_power < min_sig_cap ) ? min_sig_cap : sig_power );
+        //Capping extremely high signal to hordes
+        sig_power = ( ( sig_power > max_sig_cap ) ? max_sig_cap : sig_power );
```

Comment:
It's clearer with `std::min` and `std::max`, like `sig_power = std::min( sig_power, max_sig_cap )`.

Also, the comment above suggests that the cap is soft (ie. that sig can sometimes go above it). Does it still do that? If not, the comment should be changed.